apps/win32/wdio.conf.js

```diff
@@ -52,10 +52,10 @@ exports.config = {
 
   framework: 'jasmine',
   jasmineNodeOpts: {
-    defaultTimeoutInterval: 10000
+    defaultTimeoutInterval: 45000
```

Comment:
Maybe define these numbers on top of file instead of hard coding in numbers here?

Reply:
Changed in newest commit!